openreview/conference/templates/programchairWebfield.js

```diff
@@ -1453,10 +1453,10 @@ var displayAreaChairsStatusTable = function() {
         return row.summary.completedRecs === 0;
       },
       'msg-unsubmitted-reviews': function(row) {
-        return row.reviewProgressData.numCompletedReviews === 0;
+        return row.reviewProgressData.numCompletedReviews === 0 && row.reviewProgressData.numPapers > 0;
```

Comment:
I think this should be:

```
return row.reviewProgressData.numCompletedReviews < row.reviewProgressData.numPapers
```

A reviewer can have 2 papers assigned and entered one review and we should remind them.

Reply:
Ah you're totally right